### ISSUE TITLE
Virus warning

### ISSUE THREAD

**Issue by 7heMech** [tomblchr/atem-streamlia#9]
When I try to download app Windows blocks it for being a virus.

**Comment by tomblchr** [comment]
Thank you for your interest in this project. Are you downloading the ZIP file from the Releases? After downloading the ZIP file, extract to the contents to a folder, and using a PowerShell or CMD terminal, run SwitcherServer.exe. At what point are you being blocked exactly?

**Comment by 7heMech** [comment]
I'm blocked when the zip is scanned.

**Comment by tomblchr** [comment]
Is Windows Defender scanning the zip file, or some other virus scanner?

**Comment by 7heMech** [comment]
Maybe I should compile from source?

**Comment by tomblchr** [comment]
Absolutely, the prerequisites are .NET SDK, NodeJS and the ATEM software. https://github.com/tomblchr/atem-streamlia#building

**Comment by 7heMech** [comment]
> Absolutely, the prerequisites are .NET SDK, NodeJS and the ATEM software. https://github.com/tomblchr/atem-streamlia#building

Ooh, nodejs and c# both? Interesting.

**Comment by 7heMech** [comment]
> 
> Is Windows Defender scanning the zip file, or some other virus scanner?

Yeah, Windows Defender

**Comment by tomblchr** [comment]
Windows Defender might flag a clean zip file as containing a virus for several reasons.  If a zip file or its contents lack sufficient "reputation" data (meaning Microsoft hasn't seen enough clean instances of these files), Defender may err on the side of caution and flag it as potentially malicious. The value of open source is being able to see the code and create one's own releases as an option.
